frontend/src/components/Diff/Diff.tsx

```diff
@@ -52,23 +52,24 @@ function FormatDiffText({ texts, highlighter }: {
     highlighter: Highlighter
 }) {
     let index = 0
-    return <> {
+    return <>{
         texts.map(t =>
             Array.from(t.text.matchAll(RE_TOKEN)).map(match => {
-                const text = match[0], isToken = !match[1]
-                const key = index++
-                let className = undefined
+                const text = match[0]
+                const isToken = !match[1]
+                const key = index++ // FIXME: using index as key is an antipattern
```

Comment:
(I disagree in this case since the asm doesn't change dynamically, and there isn't any other key to use.)